test/TestConfigData/data-with-env.yaml.tmpl

```diff
@@ -0,0 +1,4 @@
+---
+data: {{ .data }}
+test-data-from-env: {{ .KD_TEST_DATA }}
+PATH: {{ .PATH }}
```

Comment:
@marcinc Nice one, good catch. I've fixed the bug and added a test to catch it too (don't replace data from environment with same name).